From 6791495ea75727be00dd54bf3bfda4e0ca3f0cb9 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 7 May 2019 19:50:28 +0200
Subject: [PATCH] [DNM] Squash of lineage-16.0-android-9.0.0_r37

Squashed commit of the following:

commit 30362305d450145e435b07642cdb3b9d42a1b461
Merge: e844dc4 b409ec2
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Feb 20 04:00:21 2019 +0000

    Snap for 5322077 from b409ec24ddcea2d744814d28ea906e20b19ad283 to pi-qpr3-release

    Change-Id: Ib4596f2f3c2977cd12f7dcd33627d487ad3b1244

commit b409ec24ddcea2d744814d28ea906e20b19ad283
Author: Bill Yi <byi@google.com>
Date:   Sun Feb 17 20:13:44 2019 -0800

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I1d7f6dc0656ae535c5a9a7e6c649e8a13e8e2fe2

Change-Id: If26ada74aed1f08d2c623ed042848e6099da6a58
---
 .../frameworks/base/core/res/res/values-en-rXC/strings.xml    | 2 +-
 overlay/frameworks/base/core/res/res/values-ta/strings.xml    | 2 +-
 .../base/packages/SystemUI/res/values-en-rXC/config.xml       | 4 ++--
 3 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/overlay/frameworks/base/core/res/res/values-en-rXC/strings.xml b/overlay/frameworks/base/core/res/res/values-en-rXC/strings.xml
index eec9afa..1773c3b 100644
--- a/overlay/frameworks/base/core/res/res/values-en-rXC/strings.xml
+++ b/overlay/frameworks/base/core/res/res/values-en-rXC/strings.xml
@@ -16,5 +16,5 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="accessibility_shortcut_toogle_warning" msgid="7758891516165017413">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‏‏‎‏‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‏‏‏‎‏‎‏‏‎‏‎‎‎‏‏‏‏‎‏‎‏‏‏‎‎‏‏‎‏‏‏‏‏‏‏‏‎‎‏‏‏‎‏‏‎‎‏‏‏‏‎‏‎‎‎‏‎‏‎When the shortcut is on, pressing both the back and down buttons for 3 seconds will start an accessibility feature.‎‏‎‎‏‏‎\n‎‏‎‎‏‏‏‎‎‏‎‎‏‏‎\n‎‏‎‎‏‏‏‎ Current accessibility feature:‎‏‎‎‏‏‎\n‎‏‎‎‏‏‏‎ ‎‏‎‎‏‏‎<xliff:g id="SERVICE_NAME">%1$s</xliff:g>‎‏‎‎‏‏‏‎‎‏‎‎‏‏‎\n‎‏‎‎‏‏‏‎‎‏‎‎‏‏‎\n‎‏‎‎‏‏‏‎ You can change the feature in Settings &gt; Accessibility.‎‏‎‎‏‎"</string>
+    <string name="accessibility_shortcut_toogle_warning" msgid="7758891516165017413">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‎‎‏‎‏‏‏‏‏‏‏‏‎‏‎‏‏‏‎‏‎‏‏‎‏‎‎‎‏‏‏‏‎‏‎‏‏‏‎‎‏‏‎‏‏‏‏‏‏‏‏‎‎‏‏‏‎‏‏‎‎‏‏‏‏‎‏‎‎‎‏‎‏‎When the shortcut is on, pressing both the back and down buttons for 3 seconds will start an accessibility feature.‎‏‎‎‏‏‎\n‎‏‎‎‏‏‏‎‎‏‎‎‏‏‎\n‎‏‎‎‏‏‏‎ Current accessibility feature:‎‏‎‎‏‏‎\n‎‏‎‎‏‏‏‎ ‎‏‎‎‏‏‎<xliff:g id="SERVICE_NAME">%1$s</xliff:g>‎‏‎‎‏‏‏‎‎‏‎‎‏‏‎\n‎‏‎‎‏‏‏‎‎‏‎‎‏‏‎\n‎‏‎‎‏‏‏‎ You can change the feature in Settings &gt; Accessibility.‎‏‎‎‏‎"</string>
 </resources>
diff --git a/overlay/frameworks/base/core/res/res/values-ta/strings.xml b/overlay/frameworks/base/core/res/res/values-ta/strings.xml
index cc05310..44a6d6f 100644
--- a/overlay/frameworks/base/core/res/res/values-ta/strings.xml
+++ b/overlay/frameworks/base/core/res/res/values-ta/strings.xml
@@ -16,5 +16,5 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="accessibility_shortcut_toogle_warning" msgid="7758891516165017413">"குறுக்குவழி அம்சம் இயக்கத்தில் இருக்கும்போது, பின்செல் பொத்தான் மற்றும் ஒலியைக் குறைக்கும் பொத்தான் இரண்டையும் 3 வினாடிகள் அழுத்திப் பிடித்தால், அணுகல்தன்மை அம்சம் இயக்கப்படும்.\n\n தற்போதைய அணுகல்தன்மை அம்சம்:\n <xliff:g id="SERVICE_NAME">%1$s</xliff:g>\n\n அமைப்புகள் &gt; அணுகல்தன்மை என்பதற்குச் சென்று அம்சத்தை மாற்றலாம்."</string>
+    <string name="accessibility_shortcut_toogle_warning" msgid="7758891516165017413">"ஷார்ட்கட் அம்சம் இயக்கத்தில் இருக்கும்போது, பின்செல் பொத்தான் மற்றும் ஒலியைக் குறைக்கும் பொத்தான் இரண்டையும் 3 வினாடிகள் அழுத்திப் பிடித்தால், அணுகல்தன்மை அம்சம் இயக்கப்படும்.\n\n தற்போதைய அணுகல்தன்மை அம்சம்:\n <xliff:g id="SERVICE_NAME">%1$s</xliff:g>\n\n அமைப்புகள் &gt; அணுகல்தன்மை என்பதற்குச் சென்று அம்சத்தை மாற்றலாம்."</string>
 </resources>
diff --git a/overlay/frameworks/base/packages/SystemUI/res/values-en-rXC/config.xml b/overlay/frameworks/base/packages/SystemUI/res/values-en-rXC/config.xml
index f1dadfb..239608f 100644
--- a/overlay/frameworks/base/packages/SystemUI/res/values-en-rXC/config.xml
+++ b/overlay/frameworks/base/packages/SystemUI/res/values-en-rXC/config.xml
@@ -17,7 +17,7 @@
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
   <string-array name="recents_tv_blacklist_array">
-    <item msgid="9050240242608018713">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‏‏‎‏‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‏‏‎‎‏‏‎‎‎‏‏‏‎‏‎‎‏‏‎‎‎‏‏‎‎‏‏‏‎‏‎‎‎‏‎‎‏‏‎‎‎‏‎‎‏‎‏‎‏‎‎‎‏‏‎‎‏‎com.google.android.tv.remote.service‎‏‎‎‏‎"</item>
-    <item msgid="8901636675968723687">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‏‏‏‏‎‏‎‎‏‎‏‏‏‏‏‏‏‏‏‏‎‏‏‏‎‎‎‏‎‎‎‏‏‏‏‎‏‏‏‎‏‏‎‎‏‎‎‏‏‎‏‎‏‏‎‏‎‏‏‎‏‎‎‎‏‏‎‏‎‏‎‏‏‏‎‎‏‏‏‎com.google.android.apps.mediashell‎‏‎‎‏‎"</item>
+    <item msgid="9050240242608018713">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‎‎‏‎‏‏‏‏‏‏‏‏‏‏‏‎‏‏‎‎‏‏‎‎‎‏‏‏‎‏‎‎‏‏‎‎‎‏‏‎‎‏‏‏‎‏‎‎‎‏‎‎‏‏‎‎‎‏‎‎‏‎‏‎‏‎‎‎‏‏‎‎‏‎com.google.android.tv.remote.service‎‏‎‎‏‎"</item>
+    <item msgid="8901636675968723687">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‏‏‏‏‎‏‎‎‏‎‏‏‏‏‏‏‏‏‏‏‎‏‏‏‎‎‎‏‎‎‎‏‏‏‏‎‏‏‏‎‏‏‎‎‏‎‎‏‏‎‏‎‏‏‎‏‎‏‏‎‏‎‎‎‏‏‎‏‎‏‎‏‏‏‎‎‏‏‏‎com.google.android.apps.mediashell‎‏‎‎‏‎"</item>
   </string-array>
 </resources>
-- 
2.17.1

